assets/wizards/advertising/index.js

```diff
@@ -6,18 +6,23 @@
  * WordPress dependencies
  */
 import { Component, render, Fragment } from '@wordpress/element';
-import { ExternalLink, SVG, Path } from '@wordpress/components';
+import { ExternalLink } from '@wordpress/components';
 import apiFetch from '@wordpress/api-fetch';
 import { __ } from '@wordpress/i18n';
 
 /**
- * Internal dependencies
+ * WordPress dependencies.
+ */
+import FeaturedVideoIcon from '@material-ui/icons/FeaturedVideo';
```

Comment:
Suggestion: use the same name for icon imports everywhere so that an icon change would only affect one line of code:

```
import HeaderIcon from '@material-ui/icons/Forum';
```

Then in all screens:

```
headerIcon={ <HeaderIcon /> }
```

Reply:
Good idea.
Fixed in https://github.com/Automattic/newspack-plugin/pull/356/commits/669542146b5bb664ba3766bc569ac5a7ce5a7f74